packages/storybook/stories/Localization.stories.mdx

```diff
@@ -40,11 +40,91 @@ For the example below, we will localize buttons in a `ControlBar` in French (Fra
 
 ### Composites
 
-If you want to localize your composite, assign a `ComponentLocale` to the `locale` prop of the composite. You may import one of our locales like `COMPONENT_LOCALE_FR_FR` for French (France)
-in the example below.
+If you want to localize your composite, assign a `ComponentLocale` to the `locale` prop of the composite. You may import one of our composite locales like
```

Comment:
I don't understand this line. Might be easier to understand with a code sample.

Reply:
Simplified the wording on the description a bit. But there is an sample after the description, although it is quite lengthy.